build.sh

```diff
@@ -27,6 +27,7 @@
 # LATEST=true             # bool - optional
 # CACHE=                  # bool - optional
 # NO_TEST=                # bool - optional
+# NO_PULL=                # bool - optional
```

Comment:
`NO_PULL` should fall under `update` variables.

Reply:
See https://github.com/startersclan/docker-sourceservers/pull/112#discussion_r1125576877